src/identify.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Wrapper for a user properties JSON object that supports operations
+ */
+
+var AMP_OP_ADD = '$add';
+var AMP_OP_SET = '$set';
+var AMP_OP_SET_ONCE = '$setOnce';
+var AMP_OP_UNSET = '$unset';
+
+
+var Identify = function() {
```

Comment:
Add documentation indicating that only the first operation of each type is recorded